app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseCategoryResource.php

```diff
@@ -63,6 +65,16 @@ public static function getPages(): array
             'create' => CreateKnowledgeBaseCategory::route('/create'),
             'view' => ViewKnowledgeBaseCategory::route('/{record}'),
             'edit' => EditKnowledgeBaseCategory::route('/{record}/edit'),
+            'subcategories' => ManageSubCategories::route('/{record}/subcategories'),
         ];
     }
+
+    public static function getRecordSubNavigation(Page $page): array
```

Comment:
I don't believe this will be needed once you have gotten rid of the manage relations page.